dashboard/slow-queries/list.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 访问慢查询列表页面
+category: how-to
+---
+
+# 访问慢查询列表页面
+
+查询集群中所有慢查询语句，系统默认将执行时间超过 Session 变量 [tidb_slow_log_threshold](https://pingcap.com/docs-cn/stable/tidb-specific-system-variables/#tidb_slow_log_threshold) 或者配置参数 [slow-threshold](https://pingcap.com/docs-cn/stable/tidb-configuration-file/#slow-threshold) 值的 SQL 语句视为慢查询语句，系统会自己记录到[慢查询日志](https://pingcap.com/docs-cn/stable/identify-slow-queries/) 中。此功能通过 SQL 将查询结果输出到页面。
```

Comment:
Do we need to use something like `/tidb-specific-system-variables.md#tidb_slow_log_threshold`? @lilin90

Reply:
@breeswish Yes. We need to use a relative path for this.